bot/constants.py

```diff
@@ -5,7 +5,7 @@
 # Lists for administration
 
 ADMIN_ROLES = ("Root", "Sudo")
-BANNED_DOMAINS = ["discord.gg"]
+BANNED_DOMAINS = []
```

Comment:
mate thats not a temp mute fix

Reply:
NO BECAUSE WE DOn'T neEd T O B L O C K  D I S C O R D  I N V I T E  L I N K S